packages/forms-web-app/src/pages/cookies/controller.test.js

```diff
@@ -11,12 +11,23 @@ const { getCookiesURL } = require('./_utils/get-cookies-url');
 const view = 'cookies/view.njk';
 const cookiesUpdatedMessagePath = 'cookies/_includes/cookies-updated-successfully-message.njk';
 const cookiesURL = getCookiesURL();
+const { mockI18n } = require('../_mocks/i18n');
 
 jest.mock('../../../src/config');
 jest.mock('../../../src/lib/remove-unwanted-cookies');
 jest.mock('../../../src/lib/flash-message');
 jest.mock('../../../src/lib/get-previous-page-path');
 
+const cookiesTranslation_EN = {
+	cookies: {
+		successBanner: {
```

Comment:
I think it would be more consistent to the tests if we were importing and using the translations json for the i18n mock function

Reply:
Sounds good, I've now updated the test to import the JSON file instead of mocking it.